docs/content/en/docs/clustermgmt/networking/cluster-replace-cilium.md

```diff
@@ -11,6 +11,7 @@ For more information on CNI customization see [Use a custom CNI]({{< ref "../../
 
 {{% alert title="Note" color="primary" %}}
 When replacing EKS Anywhere Cilium with a custom CNI, it is your responsibility to manage the custom CNI, including version upgrades and support.
+Operators can also leverage the CNI chaining feature from Isovalent where in both Cilium as the CNI and another CNI can work in a chain mode (https://docs.cilium.io/en/v1.13/installation/cni-chaining/)
```

Comment:
```suggestion
Operators can also leverage the CNI chaining feature from Isovalent where in both Cilium as the CNI and another CNI can work in a chain mode (https://docs.cilium.io/en/v1.13/installation/cni-chaining/).
```

Reply:
Looks like this is duplicated from [PR #6215](https://github.com/aws/eks-anywhere/pull/6215).

Reply:
This PR hasn't been rebased on the previous yet. Github doesn't deal with PR stacks very well.